### PR TITLE
Update requirements for supported SDK

### PR DESCRIPTION
**Pull request by JoelPasvolsky**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,4 @@
-# Replace the following with something like dwave-ocean-sdk>=6.4.2
-dimod==0.12.12
-dwave-networkx==0.8.14
-dwave-samplers==1.0.0
-dwave-system==1.19.0
+dwave-ocean-sdk>=6.6.0
 
 notebook~=7.0.0
```
Comment (randomir) [inline requirements.txt:3]: Might as well lower-bound this.

File: requirements.txt [hunk 2]
```diff
@@ -1,8 +1,4 @@
-# Replace the following with something like dwave-ocean-sdk>=6.4.2
-dimod==0.12.12
-dwave-networkx==0.8.14
-dwave-samplers==1.0.0
-dwave-system==1.19.0
+dwave-ocean-sdk>=6.6.0
```
Comment (randomir) [inline requirements.txt:1]: It doesn't really need to be as high as 6.6, right?

Comment (JoelPasvolsky) [reply requirements.txt:1]: It needs [dimod 12.12](https://github.com/dwavesystems/dwave-ocean-sdk/commit/75e3b2eef75efab4d3d2143c8514efd72e3517f8) for the wireless generators

File: requirements.txt (hunk 1 above)
Comment (JoelPasvolsky) [reply requirements.txt:3]: How about I change to `notebook~=7.0` consistent with [other ones](https://github.com/dwave-examples/reverse-annealing-notebook/blob/master/requirements.txt)?

Comment (randomir) [reply requirements.txt:3]: `~=7.0` makes more sense, yes.

File: requirements.txt (hunk 2 above)
Comment (randomir) [reply requirements.txt:1]: So it's ocean 5.5. Ok, that's pretty close.

Comment (JoelPasvolsky) [reply requirements.txt:1]: > So it's ocean 5.5. Ok, that's pretty close.

I think you mean 6.5, then yes

**Review by randomir — APPROVED** [review]

**Pull request merged**